docs/proposals/14-wireguard-based-cluster-vpn.md

```diff
@@ -0,0 +1,178 @@
+
```

Comment:
Can we rename to `GEP-15`? `GEP-14` was already proposed in https://github.com/gardener/gardener/pull/3067.
Just to avoid confusion...

Reply:
GEP-14 was not merged, so I think it's fine to keep it as it is.

Reply:
Ok, fine.